### DISCUSSION TITLE
Defining the voltage limits

### DISCUSSION THREAD

**Discussion by Akila1993** [pybamm-team/PyBaMM#2808] [Q&A]
How can we observe the values for upper and lower voltage limits for a given model (Eg: DFN , SPM etc.). 

If we want to change the voltage limits what are parameters we need to change and how we can change the limits.

**Comment by RuiheLi** [comment]
Hi, the related parameter is [here](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/input/parameters/lithium_ion/OKane2022.py#:~:text=%22Lower%20voltage%20cut,%3A%204.2%2C). This [folder ](https://github.com/pybamm-team/PyBaMM/tree/develop/examples/notebooks/parameterization)tells how to change parameters.
